stable/kommander/templates/grafana/hooks-home-dashboard.yaml

```diff
@@ -1,116 +1,41 @@
-# this is a regular Job. The retries in the configmap script
-# should ensure that this successfully runs once the Grafana
-# server is up.
 apiVersion: batch/v1
 kind: Job
 metadata:
-  name: {{ .Values.grafana.hooks.jobName | quote }}
-  namespace: {{ .Release.Namespace }}
-  labels:
-{{ include "kommander.labels" . | indent 4 }}
-spec:
-  template:
-    metadata:
-      name: {{ .Values.grafana.hooks.jobName | quote }}
-    spec:
-      restartPolicy: Never
-      containers:
-        - name: {{ .Values.grafana.hooks.jobName | quote }}
-          image: {{ .Values.grafana.hooks.image | quote }}
-          command: ["/bin/sh", "-c", "/job/run.sh"]
-          env:
-            - name: X_FORWARDED_USER
-              valueFrom:
-                secretKeyRef:
-                  name: {{ .Values.grafana.hooks.secretKeyRef }}
-                  key: username
-          volumeMounts:
-            - mountPath: /job
-              name: job
-      volumes:
-        - name: job
-          configMap:
-            name: {{ .Values.grafana.hooks.jobName }}
-            defaultMode: 0777
----
-apiVersion: batch/v1
-kind: Job
-metadata:
-  name: {{ .Values.grafana.hooks.jobName }}-upgrade-hook
+  name: {{ .Values.grafana.hooks.jobName }}
   namespace: {{ .Release.Namespace }}
   labels:
 {{ include "kommander.labels" . | indent 4 }}
   annotations:
-    helm.sh/hook: post-upgrade
-    helm.sh/hook-weight: "-4"
-    helm.sh/hook-delete-policy: before-hook-creation,hook-succeeded
+    "helm.sh/hook": post-install,post-upgrade
+    "helm.sh/hook-weight": "5"
+    "helm.sh/hook-delete-policy": before-hook-creation,hook-succeeded
 spec:
   template:
     metadata:
-      name: {{ .Values.grafana.hooks.jobName }}-upgrade-hook
+      name: {{ .Values.grafana.hooks.jobName }}
     spec:
       restartPolicy: OnFailure
       containers:
-        - name: {{ .Values.grafana.hooks.jobName }}-upgrade-hook
+        - name: {{ .Values.grafana.hooks.jobName }}
           image: {{ .Values.grafana.hooks.image | quote }}
-          command: ["/bin/sh", "-c", "/job/run.sh"]
           env:
             - name: X_FORWARDED_USER
               valueFrom:
                 secretKeyRef:
                   name: {{ .Values.grafana.hooks.secretKeyRef }}
                   key: username
-          volumeMounts:
-            - mountPath: /job
-              name: job
-      volumes:
-        - name: job
-          configMap:
-            name: {{ .Values.grafana.hooks.jobName }}
-            defaultMode: 0777
----
-apiVersion: v1
-kind: ConfigMap
-metadata:
-  name: {{ .Values.grafana.hooks.jobName }}
-data:
-  run.sh: |-
-    #!/bin/bash
-    set -o nounset
-    set -o errexit
-    set -o pipefail
-    CURL="curl --verbose --fail --max-time 30 --retry 20 --retry-connrefused"
-    DASHBOARD_ID=$($CURL -H "X-Forwarded-User: $X_FORWARDED_USER" {{ .Values.grafana.hooks.serviceURL }}/api/search/?query={{ .Values.grafana.hooks.dashboardName | urlquery }} | jq '.[0].id')
```

Comment:
I made a change here while this PR was open to go back to using the dashboard name to query for the dashboard ID - let's get that change in here as well